copilot_proxy/app.py

```diff
@@ -8,7 +8,7 @@
 from utils.codegen import CodeGenProxy
 
 codegen = CodeGenProxy(
-    host=os.environ.get("TRITON_HOST", "localhost"),
+    host=os.environ.get("TRITON_HOST", "triton"),
```

Comment:
I think that the "localhost"  is not wrong hostname because  most Linux distributions  includes the name in  **/etc/hosts** by default.

Reply:
First, this is executed in Docker, so I don't think we need to care about how it works in other distributions.

Second, [`"triton"` used to be the default here](https://github.com/moyix/fauxpilot/pull/49/commits/8895b742384181ad8abc6234029f5f95bfedbcc7#diff-8918376357384f100a61b925f3eb96ebcd3c77562cf10c530628d571919a9704L242) before #49 (file: `copilot_proxy/app.py`, line: 242), and I just reverted it back to what it was.

Of course, the last thing to point out is as described in this PR, the value `"localhost"` does not work properly. And container `triton` should be contacted at `triton:8001` within container `copilot_proxy`, otherwise error `failed to connect to all addresses` will occur.

Reply:
Yes, I think making triton the default is correct here.